.github/workflows/test.yml

```diff
@@ -54,13 +54,29 @@ jobs:
           sudo apt update
           sudo apt install -y task-spooler time
           tsp -S 48
+
       - uses: actions/checkout@v4
 
       - name: Install Python dependencies
         run: |
           . /home/firedrake/firedrake/bin/activate
           python3 -m pip install .[testing,optimisation]
 
+      - name: Check out repository linked to Trim et al. (2023)
+        uses: actions/checkout@v4
+        with:
+          repository: seantrim/exact-thermochem-solution
+          path: ./flib
+
```

Comment:
Just flagging this because I'm not sure what's right and wrong here. Do we really want to be checking out this repository every time? Would it not be easier (particularly for regression testing) to copy the current version to our test suite and make its original source clear?

Reply:
I do not have an answer to that, sadly. Could you clarify what you mean by easier?

Reply:
Something to note here is that I believe this repository only exists to support the associated publication. Essentially, it can be seen as static.